doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -218,16 +218,20 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
 ----
 
-``--acpidev_pt <HID>``
+``--acpidev_pt <HID>,<UID>``
    This option is to enable ACPI device passthrough support. The ``HID`` is a
    mandatory parameter for this option which is the Hardware ID of the ACPI
    device.
 
+   The ``UID`` is a option and used for distingnish several instances of same
+   hid devices.
```

Comment:
A couple questions: 
1. Is UID optional? If so, should we add brackets like this: ``--acpidev_pt <HID>[,<UID>]``
2. I don't understand the description of UID. Can a HID have several instances, and you need to specify one of them? Perhaps the sentence should be something like: _The ``UID`` is an option and used to specify a particular instance of the HID device._

Reply:
Yes, a HID could have several instances.
"Is UID optional? If so, should we add brackets like this: --acpidev_pt <HID>[,<UID>]"
--Our code could get HID use 00 as default UID, whether changed to "-- acpidev_pt < hid > [, < uid >]" depends on whether we want to expose it.

Reply:
What does the code do today? Does it require the user to provide a HID? What about the UID? My understanding is that the `HID` parameter is mandatory and `UID` is optional today. If true, Amy's point is that this line should read:
```
--acpidev_pt <HID>[,<UID>]
```
And we should also use the text she provided in her comment, i.e.: *The UID is an option and used to specify a particular instance of the HID device.*

Reply:
And if the default value is 00, we should add it to the description, i.e., _The UID is an option and used to specify a particular instance of the HID device. The default is 00._

Reply:
> What does the code do today? Does it require the user to provide a HID? What about the UID? My understanding is that the `HID` parameter is mandatory and `UID` is optional today. If true, Amy's point is that this line should read:
> 
> ```
> --acpidev_pt <HID>[,<UID>]
> ```
> 
> And we should also use the text she provided in her comment, i.e.: _The UID is an option and used to specify a particular instance of the HID device._

Yes, user could only set HID.

Reply:
> And if the default value is 00, we should add it to the description, i.e., _The UID is an option and used to specify a particular instance of the HID device. The default is 00._

OK.